recipes/xtcocotools/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "xtcocotools" %}
+{% set version = "1.14.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 6434382784d206204d2a1b491dba359ff3464a2b9eb5890fbc0d868cad4e810f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cython
+  host:
+    - cython
+    - matplotlib
+    - numpy >=1.20.0
```

```suggestion
    - numpy
```
Don't pin so it uses the global pin